tests/test_brm.py

```diff
@@ -999,73 +987,72 @@ def test_coef_names(formula_str, non_real_cols, expected_names):
 
 
 @pytest.mark.parametrize('fitargs', [
+    lambda S: dict(backend=pyro_backend, algo='prior', num_samples=S),
     lambda S: dict(backend=pyro_backend, iter=S, warmup=0),
     lambda S: dict(backend=pyro_backend, iter=S // 2, num_chains=2, warmup=0),
     lambda S: dict(backend=pyro_backend, algo='svi', iter=1, num_samples=S),
-    lambda S: dict(backend=pyro_backend, algo='prior', num_samples=S),
-    # Set environment variable `RUN_SLOW=1` to run against the NumPyro
-    # back end.
-    pytest.param(
-        lambda S: dict(backend=numpyro_backend, iter=S, warmup=0),
-        marks=pytest.mark.skipif(not os.environ.get('RUN_SLOW', ''), reason='slow')),
-    pytest.param(
-        lambda S: dict(backend=numpyro_backend, iter=S // 2, num_chains=2, warmup=0),
-        marks=pytest.mark.skipif(not os.environ.get('RUN_SLOW', ''), reason='slow')),
-    pytest.param(
-        lambda S: dict(backend=numpyro_backend, algo='prior', num_samples=S),
-        marks=pytest.mark.skipif(not os.environ.get('RUN_SLOW', ''), reason='slow')),
+    lambda S: dict(backend=pyro_backend, algo='svi', iter=1, num_samples=S, subsample_size=1),
+    lambda S: dict(backend=numpyro_backend, algo='prior', num_samples=S),
+    lambda S: dict(backend=numpyro_backend, iter=S, warmup=0),
```

Comment:
numpyro should generally be much faster to sample from but I think that with such small number of samples, the compilation overhead dominates. Let us know if you see any examples where the compilation time seems unreasonably high.

Reply:
Yes, this `RUN_SLOW` stuff was just about avoiding the compilation time of nuts. This seems tolerable for regular usage, but quickly adds up in tests.